scripts/chce_jenkins.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+# Jenkins na mikrusowym porcie 
+# Autor: Maciej Loper
+
+status() {
+    echo "[x] $1"
+}
+
+status "dodawanie repozytorium Jenkinsa"
+wget -q -O - https://pkg.jenkins.io/debian-stable/jenkins.io.key | sudo apt-key add -
+sudo sh -c 'echo deb http://pkg.jenkins.io/debian-stable binary/ > /etc/apt/sources.list.d/jenkins.list'
+
+status "aktualizacja repozytoriow"
+sudo apt update
+echo
+
+status "instalacja Jenkinsa i Javy JRE11"
+sudo apt install -y openjdk-11-jre-headless jenkins
```

Comment:
```suggestion
sudo apt install -y openjdk-11-jre-headless
sudo apt install -y jenkins
```

Uniknięcie błędów z uruchomieniem jenkinsa, jako że pierwsze Java powinna być skonfigurowana.